docs/user-guide/tools/mono_dl2.rst

```diff
@@ -0,0 +1,181 @@
+.. _mono_dl2:
+
+***************************************************************
+How to do monoscopic DL2 reconstruction using ``ctapipe`` tools
+***************************************************************
+
+This guide explains how to use the machine learning capabilities of ``ctapipe``
+to process files containing image parameters (data level 1b/ DL1b) to data level 2 (DL2).
+
+.. NOTE::
+   * This guide only shows how to perform mono reconstruction, but stereo reconstruction
+     is also supported by ``ctapipe``.
```

Comment:
How would this guide be different for the stereo case?

Reply:
Ah, right, I went through it too fast. Maybe just indicate the difference in that sentence standard/stereo `HillasReconstructor` vs Disp method for mono.